tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/workspace/NewLiferayWorkspaceWizard.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
```

Comment:
For new module project wizard lets use large Liferay Icon as a wizard banner image.